daml-lf/spec/daml-lf-1.rst

```diff
@@ -390,16 +390,17 @@ and other similar pitfalls. ::
 We can now define a generic notion of *identifier* and *name*::
 
   identifiers:
-          Ident  ∈  [a-zA-Z_\$][a-zA-Z0-9_\$]       -- Ident
+          Ident  ∈  [a-zA-Z_\$][a-zA-Z0-9_\$]{999}  -- Ident
```

Comment:
```suggestion
          Ident  ∈  [a-zA-Z_\$][a-zA-Z0-9_\$]{1000}  -- Ident
```

Reply:
or limit to 999 but currently your limit seems to be 1000 while you specify 999

Reply:
There is indeed an error here, but not the one you pointed out. 
The correct regexp is 
```
[a-zA-Z_\$][a-zA-Z0-9_\$]{0,999} 
```
or in full English: An `Identifier` is a non empty string starting with one character matching `[a-zA-Z_\$]`  followed by 0 to 999 character matching `[a-zA-Z0-9_\$]`.

Reply:
`1 + 999 = 1000`

Reply:
duh ofc, I need more coffee